packages/hooks/src/useEventListener.ts

```diff
@@ -40,14 +40,34 @@ export default function useEventListener<Type extends keyof DocumentEventMap>(
       return;
     }
 
+    // @ts-expect-error Handle this in case non-TypeScript users pass in the wrong value
+    if (enabled === 'false') {
+      console.error(
+        'The string "false" was received for property `enabled`. A boolean value was expected.',
+      );
+      return;
+    }
+
     const callback = (e: DocumentEventMap[Type]) =>
       memoizedEventCallback.current(e);
 
+    const eventListenerOptions = {
+      ...options,
+      once: enabled === 'once',
+    };
     // NOTE(JeT): I'm pretty sure there should be a way to avoid this type assertion, but I couldn't figure it out.
-    element.addEventListener(type, callback as EventListener, options);
+    element.addEventListener(
+      type,
+      callback as EventListener,
```

Comment:
You can't figure this out @lazytype ???!?!!? :P

Reply:
I did look into it actually :P. the typing that TS provides for `addEventListener` isn't intelligent enough. not motivated enough at the moment to get it changed upstream